backend/vkt/src/main/java/fi/oph/vkt/util/exception/APIExceptionType.java

```diff
@@ -11,6 +11,7 @@ public enum APIExceptionType {
   INITIALISE_ENROLLMENT_IS_FULL,
   INITIALISE_ENROLLMENT_REGISTRATION_CLOSED,
   INITIALISE_ENROLLMENT_TO_QUEUE_HAS_ROOM,
+  INITIALISE_ENROLLMENT_DUPLICATE_PERSON,
```

Comment:
Voi jäädä näinkin mut jos viitsit niin tän ja frontin api errorin sekä lokalisoinnin voi aakkosjärjestää. Näin jos/kun tulee lokalisoinnit jsonissa aakkosjärjestettyä jossain kohtaa, niin näitä pystyy kätevästi sinne vertaamaan.